lib/executor/ramping_arrival_rate.go

```diff
@@ -183,16 +183,20 @@ func (varr *RampingArrivalRate) Init(ctx context.Context) error {
 // Lets look at a simple example - lets say we start with 2 events and the first stage is 5
 // seconds to 2 events/s and then we have a second stage for 5 second that goes up to 3 events
 // (using small numbers because ... well it is easier :D). This will look something like:
-//  ^
+//
+//	^
+//
 // 7|
 // 6|
 // 5|
 // 4|
 // 3|       ,-+
 // 2|----+-'  |
 // 1|    |    |
-//  +----+----+---------------------------------->
-//  0s   5s   10s
+//
+//	+----+----+---------------------------------->
+//	0s   5s   10s
```

Comment:
This looks like a mistake to be changed and likely should be changed by hand so that it makes `gofmt` happy